### PR TITLE
refactor(api): Update remaining webhook routes to modular structure

### PR DESCRIPTION
**Pull request by DafyddLlyr**
# What does this PR do?
 - Refactors remaining `/webhook` routes [to format proposed here](https://github.com/theopensystemslab/planx-new/blob/main/doc/architecture/decisions/0007-modularise-express-application.md)
 - Adds documentation
 - Updates types and tests
 - No changes to business logic, just a refactor - the only exception is to update the return type of one API endpoint (see comment)

image

### CODE REVIEW

File: api.planx.uk/hasura/metadata/index.ts [hunk 1]
```diff
@@ -25,13 +25,18 @@ type RequiredScheduledEventArgs = Pick<
   "webhook" | "schedule_at" | "comment" | "payload"
 >;
 
+export interface ScheduledEventResponse {
```
Comment (DafyddLlyr) [inline api.planx.uk/hasura/metadata/index.ts:28]: Docs: https://hasura.io/docs/latest/api-reference/metadata-api/scheduled-triggers/#metadata-create-scheduled-event

Comment (DafyddLlyr) [reply api.planx.uk/hasura/metadata/index.ts:28]: Typing this cascaded across a few functions and tests which is a bit noisy for a PR, but quite nice in practice.

File: api.planx.uk/inviteToPay/createPaymentSendEvents.ts [hunk 2]
```diff
@@ -14,9 +17,9 @@ enum Destination {
 }
 
 interface CombinedResponse {
```
Comment (DafyddLlyr) [inline api.planx.uk/inviteToPay/createPaymentSendEvents.ts:19]: This type is repeated in `api.planx.uk/send/createSendEvents.ts` but I'm ignoring this for now as this PR is already bigger than it should be.

File: api.planx.uk/modules/webhooks/service/lowcalSessionEvents/index.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  createdAt,
+  payload,
+}: CreateSessionEvent) => {
+  const response = await createScheduledEvent({
+    webhook: "{{HASURA_PLANX_API_URL}}/send-email/expiry",
+    schedule_at: addDays(createdAt, DAYS_UNTIL_EXPIRY),
+    payload: payload,
+    comment: `expiry_${payload.sessionId}`,
+  });
+  return [response];
```
Comment (DafyddLlyr) [inline api.planx.uk/modules/webhooks/service/lowcalSessionEvents/index.ts:43]: Returning `[response]` as opposed to just `response` gives us a consistent return type across all these APIs.

**Review by jessicamcinchak — APPROVED** [review]

**Pull request merged**